app/controllers/grade_entry_forms_controller.rb

```diff
@@ -162,6 +162,7 @@ def populate_grades_table
       s = student.attributes
       student_grade_entry = @grade_entry_form.grade_entry_students
                                              .find_by_user_id(student.id)
+      s[:section] = student.section.name
```

Comment:
This is only good if the student is enrolled in a section; otherwise, it'll raise an error.